docker-compose.yml

```diff
@@ -32,14 +41,24 @@ services:
     image: ckwagaba/osprey-backend:latest
     container_name: flask-api
     environment:
-      DATABASE_URI:
-      FLASK_APP_SECRET:
       KUBE_HOST:
       KUBE_TOKEN:
       KUBE_CERT:
-      MONGO_URI: mongodb://crane:password@crane-mongo-db:27017/admin
+      FLASK_APP_SECRET:
+      FLASK_APP_SALT:
+      FLASK_VERIFY_SALT:
+      APP_MAIL_USERNAME:
+      APP_MAIL_PASSWORD:
+      PYTHONDONTWRITEBYTECODE: 1
+      PYTHONUNBUFFERED: 1
+      FLASK_ENV: development
+      MONGO_URI: ${MONGO_URI:-mongodb://crane:password@crane-mongo-db:27017/admin}
+      REDIS_URL: ${REDIS_URL:-redis://crane-redis-db:6379}
+      DATABASE_USER: postgres
+      DATABASE_URI: ${DATABASE_URI:-postgresql://postgres:postgres@database:5432/cranecloud}
```

Comment:
@Mubangizi bro just to confirm, is it cool to have these out in the open like this?

Reply:
They are relative to docker compose network so.